### PR TITLE
[MODULAR] Makes the best carpet available for purchase

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

you can now buy a crate with executive carpet in it

## Why It's Good For The Game

The exec carpet is the best carpet, and the only way you can get is either it spawning in the map, or maint crawling, which is annoying. That is now fixed.

## Changelog
:cl:
add: added exec carpet crate
/:cl:

### CODE REVIEW

**Comment by Arctaisia** [comment]
Best PR,,,,

**Comment by Krysonism** [comment]
I'm happy that people enjoy my carpets. 

However I'd like to caution against making the supply of scare items infinite and easy to obtain.

I intentionally avoided adding a cargo crate for these carpets with the reasoning that rarity and scarcity is a big part of what makes something cool.

I don't play skyrat so my opinion should not be given much weight, but I think what I'd like to call "candy for breakfast design" is a big problem across all of SS13.

**Comment by GoldenAlpharex** [comment]
> I'm happy that people enjoy my carpets.
> 
> However I'd like to caution against making the supply of scare items infinite and easy to obtain.
> 
> I intentionally avoided adding a cargo crate for these carpets with the reasoning that rarity and scarcity is a big part of what makes something cool.
> 
> I don't play skyrat so my opinion should not be given much weight, but I think what I'd like to call "candy for breakfast design" is a big problem across all of SS13.

That's a fair comment, and I at first was going to make a PR about it a few months back, but I do enjoy the scarcity of it, and if anything, I think it should be a lot more expensive to make up for the fact of how rare it's meant to be.

**Comment by SpaceVampire** [comment]
Not only do I disagree with having it available for purchase at all.
But why in the world is it that cheap?
	/// Cost of the crate. DO NOT GO ANY LOWER THAN X1.4 the "CARGO_CRATE_VALUE" value if using regular crates, or infinite profit will be possible!
	var/cost = CARGO_CRATE_VALUE * 1.4
As you can see. That is the cost of just the crate, without anything in it.
So you have the most rare and fancy carpet, being sold at net loss on shipping alone.

**Comment by axietheaxolotl** [comment]
> 
> 
> I'm happy that people enjoy my carpets.
> 
> However I'd like to caution against making the supply of scare items infinite and easy to obtain.
> 
> I intentionally avoided adding a cargo crate for these carpets with the reasoning that rarity and scarcity is a big part of what makes something cool.
> 
> I don't play skyrat so my opinion should not be given much weight, but I think what I'd like to call "candy for breakfast design" is a big problem across all of SS13.

I think that's a fair point, but honestly. Making it scarce doesn't really do much, at least for me. It doesn't really make it feel more special, it just makes it more annoying that I have something I want, but it's entirely up to RNG whether or not I get it. The scarcity of it does little for me, it's just an annoyance more then anything.

**Comment by axietheaxolotl** [comment]
> 
> 
> Not only do I disagree with having it available for purchase at all.
> But why in the world is it that cheap?
> /// Cost of the crate. DO NOT GO ANY LOWER THAN X1.4 the "CARGO_CRATE_VALUE" value if using regular crates, or infinite profit will be possible!
> var/cost = CARGO_CRATE_VALUE * 1.4
> As you can see. That is the cost of just the crate, without anything in it.
> So you have the most rare and fancy carpet, being sold at net loss on shipping alone.

What I get for not reading the code, yeah it's been fixed. Bumped the price up a decent bit.

**Comment by GoldenAlpharex** [comment]
I don't see why it's not like the other carpet single-packs.

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 1]
```diff
@@ -355,6 +355,16 @@
 					/obj/item/fuel_pellet)
 	crate_name = "drone fuel crate"
 
+/datum/supply_pack/service/executive
+	name = "Executive Carpet Crate"
+	desc = "Want to show your love for the Greatest Company in the Universe? Well now you can with a supply of a comfy blue carpet, adorned with NT's beautiful logo."
+	cost = CARGO_CRATE_VALUE * 3
+	contains = list(
+		/obj/item/stack/tile/carpet/executive/thirty,
+		/obj/item/stack/tile/carpet/executive/thirty,
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cargo/code/packs.dm]:
```suggestion
```

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 2]
```diff
@@ -355,6 +355,16 @@
 					/obj/item/fuel_pellet)
 	crate_name = "drone fuel crate"
 
+/datum/supply_pack/service/executive
+	name = "Executive Carpet Crate"
+	desc = "Want to show your love for the Greatest Company in the Universe? Well now you can with a supply of a comfy blue carpet, adorned with NT's beautiful logo."
+	cost = CARGO_CRATE_VALUE * 3
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cargo/code/packs.dm]:
```suggestion
	cost = CARGO_CRATE_VALUE * 4
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
My offer on this, making it more expensive and there's less.

**Comment by GoldenAlpharex** [comment]
@axietheaxolotl Update from master to make sure the checks don't fail anymore.

**Comment by Gandalf2k15** [comment]
Given what the creator stated, let's not go down this slippery slope.

**Pull request closed without merging**